packages/react/test/evaluation.spec.tsx

```diff
@@ -577,5 +580,76 @@ describe('evaluation', () => {
         );
       });
     });
+
+    describe('HookFlagQuery',  () => {
+      it('should return errors correctly', () => {
+        const errorMessage = 'no flag found with key i-dont-exist';
+        const details: EvaluationDetails<boolean> = {
+          flagKey: 'i-dont-exist',
+          flagMetadata: {},
+          errorCode: ErrorCode.FLAG_NOT_FOUND,
+          errorMessage,
+          value: true,
+        };
+
+        let hookFlagQuery = new HookFlagQuery(details);
+        expect(hookFlagQuery.isError).toEqual(true);
+        expect(hookFlagQuery.errorCode).toEqual(ErrorCode.FLAG_NOT_FOUND);
+        expect(hookFlagQuery.errorMessage).toEqual(errorMessage);
+
+        details.errorCode = undefined;
+        details.reason = StandardResolutionReasons.ERROR;
+        hookFlagQuery = new HookFlagQuery(details);
+        expect(hookFlagQuery.isError).toEqual(true);
+      });
+
+      it('should return flag metadata', () => {
+        const flagMetadata = {
+          'ping': 'pong'
+        };
+        const details: EvaluationDetails<boolean> = {
+          flagKey: 'with-metadata',
+          flagMetadata,
+          value: true,
+        };
+        const hookFlagQuery = new HookFlagQuery(details);
+        expect(hookFlagQuery.flagMetadata).toEqual(flagMetadata);
+      });
+
+      it('should return details', () => {
+        const details: EvaluationDetails<string> = {
+          flagKey: 'flag-key',
+          flagMetadata : {},
+          value: 'string'
+        };
+
+        const hookFlagQuery = new HookFlagQuery(details);
+        expect(hookFlagQuery.details).toEqual(details);
+      });
+
+      it('should return is authorative correctly', () => {
```

Comment:
```suggestion
      it('should return is authorative if Reason != STALE/DISABLED and errorCode unset', () => {
```

I try to avoid vagueness like this in test descriptions, it's helpful to future readers if the `it` properly describes the rule.